app/modules/asset_groups/models.py

```diff
@@ -786,7 +786,6 @@ def remove_annotation_from_encounter(self, encounter_guid, annot_guid):
                     'annotations' in encounter_metadata.keys()
                     and annot_guid in encounter_metadata['annotations']
                 ):
-                    breakpoint()
```

Comment:
that should never have been there, almost certainly my fault

Reply:
i had pulled this out in one my works-in-progress.  great, now i am going to get a conflict!  lol

Reply:
saweeee